### ISSUE TITLE
you probably should change the example to not be /image/upload, but instead / so it works out of the box

### ISSUE THREAD

**Issue by cryptomail** [chmorgan/connect-busboy-example#1]
line 27 of app.js uses /image/upload but the form won't post to there unless you directly specify action appropriately.

**Comment by chmorgan** [comment]
Yeah. I was using this only to provide an example to the connect-busboy guys for an issue I was seeing. github doesn't let me hide the project without paying for an account... I *could* delete it but that seems not good either...
